books/RayTracingInOneWeekend.html

```diff
@@ -777,10 +806,10 @@
 visualize the normals with a color map. A common trick used for visualizing normals (because it’s
 easy and somewhat intuitive to assume $\mathbf{n}$ is a unit length vector -- so each
 component is between -1 and 1) is to map each component to the interval from 0 to 1, and then map
-x/y/z to r/g/b. For the normal, we need the hit point, not just whether we hit or not. We only have
-one sphere in the scene, and it's directly in front of the camera, so we won't worry about negative
-values of $t$ yet. We'll just assume the closest hit point (smallest $t$). These changes in the code
-let us compute and visualize $\mathbf{n}$:
+$(x, y, z)$ to "red-green-blue". For the normal, we need the hit point, not just whether we hit or
```

Comment:
I think "and then map $(x, y, z)$ to $(\text{red}, \text{green}, \text{blue})" would be better.